README.md

```diff
@@ -8,49 +8,231 @@ Table of Contents
 ================
 
 - [Overview](#overview)
+- [Flags at a glance](#flags-at-a-glance)
 - [Structure of data](#structure-of-data)
 - [Important flagging details](#important-flagging-details)
 - [AWS Glue job documentation](#aws-glue-job-documentation)
+- [Next steps](#next-steps)
 
 # Overview
 
 This repository contains code to identify and flag sales that may be non-arms-length transactions. A non-arms-length sale occurs when the buyer and seller have a relationship that might influence the transaction price, leading to a sale that doesn't reflect the true market value of the property. These sales can distort our analyses and models, since they don't adhere to the principle of an open and competitive market.
 
-The workflow for sale flagging is as follows:
 
-* A manual initial run of `manual_flagging/initial_flagging.py` instantiates all tables and flags all specified sales as either outliers or non-outliers.
-* Next, `glue/sales_val_flagging.py` flags all new, unflagged sales. This script is automated such that it runs on a schedule (e.g. monthly).
-* If an error occurs or we want to update the methodology on previously-flagged sales, `manual_flagging/manual_update.py` is used to select a subset of sales to re-flag. All sales have a version number that is incremented on update. When utilizing our sales views, we pull the flag data with the highest version value to keep it up-to-date.
+## Sales Validation Pipeline Run Modes
```

Comment:
**suggestion:** I would move this entire section under the **Flagging details** section. It's important, but the breakdown of what we're flagging and what the flags are is much _more_ important.